pennylane_lightning/core/src/observables/Observables.hpp

```diff
@@ -192,6 +241,15 @@ class HermitianObsBase : public Observable<StateVectorT> {
     void applyInPlace(StateVectorT &sv) const override {
         sv.applyMatrix(matrix_, wires_);
     }
+
+    void
+    applyInPlaceShots([[maybe_unused]] StateVectorT &sv,
+                      [[maybe_unused]] std::vector<size_t> &identity_wire,
+                      [[maybe_unused]] std::vector<size_t> &ob_wires,
+                      [[maybe_unused]] size_t term_idx = 0) const override {
+        PL_ABORT("For Hermitian Observables, the applyInPlaceShots method is "
+                 "not supported.");
```

Comment:
I would favour more "active"-voice messages, for instance
```suggestion
        PL_ABORT("The Hermitian observable does not support applyInPlaceShots.");
```

Reply:
Sure!